projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 12.1-a.5 - 2023-04-10
+### Enhancements
+- Implemented a "tabbed" variation for static filters. This adds tabs on top of the results for each filter group. [#29811]
```

Comment:
I would probably rephrase this so we know what feature / part of the product this is talking about, but I suppose this can be done later in the readme.txt too.

Reply:
Yes, thank you, I'll change that in a separate PR.